webpack/assets/javascripts/compute_resource/libvirt.js

```diff
@@ -1,3 +1,10 @@
+/* eslint-disable jquery/no-closest */
```

Comment:
Why disable all the jquery rules with magic comments, and not in eslint configuration?

Reply:
These comments are only in legacy js files that still use jquery.
With this approach, we would decrease the use of it over time